generate-stackbrew-library.sh

```diff
@@ -3,6 +3,7 @@ set -Eeuo pipefail
 
 declare -A aliases=(
 	[1.11]='1 latest'
+	[1.12-rc]='rc'
```

Comment:
Should these be `1.12beta1`, instead of `rc`? (I see some other locations above as well where `-rc` is used)

Reply:
We use 'rc' as a generic "pre-release" designation in folder names and tags; this should generate the beta 1 tag too.

Reply:
ah, thanks! sorry, I got confused 😊